<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nódulos Pulmonares - Fecho convexo e renderização</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>241</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Eye</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Eye_x_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>-150.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Eye_y_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>150.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Eye_z_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>-150.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Look_At</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="La_x_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="La_y_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="La_z_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>A View Up</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Av_x_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Av_y_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>490</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Av_z_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>490</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9990.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>125</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Selecione arquivo de nº :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>230</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Exportar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Importar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Fecho">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecho Convexo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configurações de Renderização</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
   </widget>
   <widget class="GLWidget" name="GLwidget" native="true">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>13</y>
      <width>551</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>do_save()</slot>
    <slot>do_load()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
